### PR TITLE
Redesign the layout vertically

### PR DESCRIPTION
**Pull request by hakimelek**
@lorensr 

* Redesigned the layout vertically. 

Review: 
* Animation.
* Book Description layout.

### CODE REVIEW

**Comment by lorensr** [comment]
image

suggestions:

- left side:
  - vertically centered
  - animate from left, not top
  - take up less than 50% width?
- right side:
  - vertically centered
  - font size is small, at least for a landing page like this with so little total text. compare to eg https://www.discovermeteor.com/ at 18 or https://stripe.com/ at 17
  - max width on the whole section
- twirl in form doesn't make as much sense now that it's not a circle. what about zoom up from bottom? so no rotate, just translate y and scale to 1
- covered technologies section starts out the page load visible before being hidden

File: pages/index.js [hunk 1]
```diff
@@ -139,253 +124,336 @@ class Index extends Component {
...
-                { // eslint-disable-next-line
```
Comment (lorensr) [inline pages/index.js]: either keep these in to prevent eslint error, or change the svgs to be compliant (like the bottom svg)

**Comment by lorensr** [comment]
and medium, the gold standard in readability, looks like 21px!

**Comment by lorensr** [comment]
a lot of repetition, use component:

image

**Comment by lorensr** [comment]
use divs instead of spans with display:block

image

**Comment by lorensr** [comment]
usually i would use either margin or padding. i default to margin unless there's an effect that needs padding, like :active link or bgcolor

image

**Comment by lorensr** [comment]
@hakimelek check out the above comments & individual commits

lmk if anyone has comments @helfer @jkrup & kinu (get her on GH! 😄)

## Before

https://graphql-guide-gnwallgdnv.now.sh 

video: https://www.dropbox.com/s/cbnx93umvcnyme6/top-header.mov?dl=0

image

## After

https://graphql-guide-sbvanonsyk.now.sh

video: https://www.dropbox.com/s/on8r48bzie7g6wk/left-header.mov?dl=0

image

**Comment by hakimelek** [comment]
Not sure why I didn't see your comments before, but thanks for the update :)

**Pull request merged**